### PR TITLE
fix: disable cache for cargo binary

### PR DESCRIPTION
**Pull request by usamoi**
Fix https://github.com/tensorchord/pgvecto.rs/actions/runs/6701671369/job/18209452281#step:5:143 .

### CODE REVIEW

**Comment by usamoi** [comment]
@silver-ymz Can you review this PR?

**Comment by silver-ymz** [comment]
> Can you review this PR?

Yes, this change is right.

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -118,13 +115,12 @@ jobs:
         wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
         sudo apt-get update
         sudo apt-get -y install libpq-dev postgresql-${{ matrix.version }} postgresql-server-dev-${{ matrix.version }}
-        cargo install cargo-pgrx --git https://github.com/tensorchord/pgrx.git --rev $(cat Cargo.toml | grep "pgrx =" | awk -F'rev = "' '{print $2}' | cut -d'"' -f1) || true
+        cargo install cargo-pgrx --git https://github.com/tensorchord/pgrx.git --rev $(cat Cargo.toml | grep "pgrx =" | awk -F'rev = "' '{print $2}' | cut -d'"' -f1)
```
Comment (silver-ymz) [inline .github/workflows/check.yml:118]:
```suggestion
        (cargo install cargo-pgrx --git https://github.com/tensorchord/pgrx.git --rev $(cat Cargo.toml | grep "pgrx =" | awk -F'rev = "' '{print $2}' | cut -d'"' -f1)) || true
```

After researching, I found that shell script must add bracket in previous command like `(command) || true` to ignore the error code.

If we want to re-enable the cache for binary, we can add bracket for each commands.

Comment (usamoi) [reply .github/workflows/check.yml:118]: Why would cargo return a status which is not 0?

Comment (silver-ymz) [reply .github/workflows/check.yml:118]: Because when the cache was successfully restored, the `cargo-pgrx` had already been downloaded.

Comment (usamoi) [reply .github/workflows/check.yml:118]: It seems that the broken action is caused by incorrect .so library version. If we re-enable cache for binaries, we may need to lock versions of system C/C++ dependencies. Do you have ideas about it?

Comment (silver-ymz) [reply .github/workflows/check.yml:118]: Since that, disabling cache for binaries may be the better way.

**Review by silver-ymz — APPROVED** [review]

**Pull request merged**